pass/disk.go

```diff
@@ -109,11 +114,11 @@ func (s *diskStore) GlobSearch(query string) ([]string, error) {
 
 		for i, match := range allMatches {
 			// TODO this does not handle identical file names in multiple s.paths
```

Comment:
Now you can remove the TODO, here and below in `Open` 🙂

Reply:
Done.